pom.xml

```diff
@@ -212,6 +213,13 @@
         <version>${version.org.assertj_assertj-core}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
Okay, but shouldn't the bom be declared in the web/ project only? The other modules do not need swarm